internal/models/changelog.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+// ChangelogEntry - JSON for individual rule from changlog table
+type ChangelogEntry struct {
+	ID           int    `json:"id"`
+	ResponseCode string `json:"response_code"`
+	EnhancedCode string `json:"enhanced_code"`
+	Regex        string `json:"regex"`
+	Priority     int    `json:"priority"`
+	Description  string `json:"description"`
+	BounceAction string `json:"bounce_action"`
+	UserID       string `json:"userid"`
+	CreatedAt    string `json:"createdat"`
+	Comment      string `json:"comment"`
+	
+}
+
+
+type ChangelogTable struct {

```

Comment:
This struct doesn't provide any additional information that `[]ChangelogEntry` does, e.g. if have a `[]ChangelogEntry`, then I can calculate `NumRules` with the `len` function. For the functions that return `ChangelogTable`, why not just have return `[]ChangelogEntry`?